src/fontra/client/web-components/menu-bar.js

```diff
@@ -120,6 +121,38 @@ export class MenuBar extends SimpleElement {
     this.contentElement.appendChild(menuPanel);
   }
 
+  handleKeyDown(event) {
+    event.stopImmediatePropagation();
+    switch (event.key) {
+      case "ArrowLeft":
+      case "ArrowRight":
+        this.navigateMenuBar(event.key);
+        break;
+    }
+  }
+
+  navigateMenuBar(arrowKey) {
+    const currentSelection = this.contentElement.querySelector(".current");
+    const menuItemElements = this.contentElement.children;
+    const currentSelectionIndex = Array.prototype.indexOf.call(
+      menuItemElements,
+      currentSelection
+    );
+    let newSelectionIndex;
+    arrowKey == "ArrowLeft"
```

Comment:
The ternary operator (`?`) is an expression itself, and one generally shouldn't use the branches for side effects (in this case: assign to a variable).

This can be more elegantly written using the `?` expression as an expression (untested):
```js
    const newSelectionIndex = currentSelectionIndex + (arrowKey == "ArrowLeft" ? -1 : +1);
```
(Not sure if the parentheses are needed, prettier will tell you...)

Reply:
This is really smart! Thank you!